cmd/ovirt-populator/ovirt-populator.go

```diff
@@ -92,7 +84,7 @@ func populate(crName, engineURL, secretName, diskID, volPath, namespace, crNames
 		diskID,
 		volPath,
 	}
-	config, err := rest.InClusterConfig()
+	/*config, err := rest.InClusterConfig()
```

Comment:
many things are comment out now. do we wish to preserve it with a comment? or we should just drop it?
mostly talking about the progress.